src/common/node/childProcess.ts

```diff
@@ -58,13 +64,13 @@ export class ChildProcess {
 
         let spawnedProcess = child_process.spawn(command, args, options);
         spawnedProcess.once("error", (error: any) => {
-            outcome.reject({ error: error });
+            outcome.reject(error);
         });
         spawnedProcess.once("exit", (code: number) => {
             if (code === 0) {
                 outcome.resolve(void 0);
             } else {
-                outcome.reject({error: code});
```

Comment:
Changed to:
outcome.reject(new Error(`Command ${command} failed with error code ${code}`));